docs/docs/contribution/new-logo.html

```
@@ -0,0 +1,1151 @@
+<head>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)